README.md

```diff
@@ -30,165 +32,568 @@ The two guiding principles are:
+
+##### Available components and functions
+
+These components and functions are available as named exports on the `hops` package.
+
+###### `render(element, [options])`
+
+This is the main render function which you must call in - and export from - your entry file. It accepts two arguments: A React element (which is your root application) and an optional options hash that contains runtime configuration.
+
+```javascript
+import { render } from 'hops';
+
+export default render(<MyApp />);
+```
+
+###### `<Miss />`
+
+This component allows you to declaratively inform the Express.js server that there is nothing to render and it should delegate to the next middleware / or return with a `404` status code.
+
+It is commonly used in an applications routing config as the last route:
 
 ```javascript
-import { render } from 'hops-react';
+import { render, Miss } from 'hops';
 import React from 'react';
+import { Route, Switch } from 'react-router-dom';
 
-export default render(<h1>Hello World!</h1>);
+const Home = () => <h1>Home</h1>;
+const About = () => <h1>About</h1>;
+
+export default render(
+  <Switch>
+    <Route exact path="/" component={Home} />
+    <Route exact path="/about" component={About} />
+    <Miss />
+  </Switch>
+);
 ```
 
-For more details and advanced use-cases read the [full readme of `hops-react`](https://github.com/xing/hops/tree/master/packages/react).
+###### `<Status code={Number} />`
+
+With this component you can declaratively change the status code of the HTTP response for server-side rendering:
+
+```javascript
+import { render, Status } from 'hops';
+import React from 'react';
+
+export default render(<Status code={418} />);
+```
+
+###### `<Header name={String} value={String} />`
+
+With this component you can specify additional HTTP headers to be sent in the server-rendered response:
+
+```javascript
+import { render, Header } from 'hops';
+import React from 'react';
+
+export default render(<Header name="X-Foo" value="my-value" />);
+```
+
+###### `importComponent(module, [exportName])`
+
+Using the `importComponent()` function you can asynchronously load modules as React components into your application to help you reduce bundle sizes.
+
+It works similarly to [`react-loadable`](https://github.com/jamiebuilds/react-loadable) but is deeply integrated with Hops to enable server-side rendering, etc.
+
+```javascript
+import { render, importComponent } from 'hops';
+
+const Home = importComponent('./home');
+
+export default render(<Home />);
+```
+
+Components created using `importComponent` support some additional props to control module loading and placeholder rendering:
+
+```javascript
+import { render, importComponent } from 'hops';
+
+const About = importComponent('./about');
+
+const loader = load =>
+  Promise.race([
+    new Promise((resolve, reject) => setTimeout(reject, 10000)),
+    load(),
+  ]);
+
+const render = ({ Component, error, loading, ...props }) => {
```

Comment:
Call this function `renderAbout`, otherwise Hops's `render` function is overwritten.